From 3ff962719efaed47b2d2bde55a5ca19285f91ccd Mon Sep 17 00:00:00 2001
From: xNombre <kartapolska@gmail.com>
Date: Fri, 23 Jul 2021 11:37:54 +0200
Subject: [PATCH] init.rc: Don't enable f2fs iostat by default

It is only intended for performance analysis and can
impact I/O throughput.

Change-Id: Ia7d3a449d52625d911e2cea16249972d01e5cfcf
Signed-off-by: Adithya <gh0strider.2k18.reborn@gmail.com>
Signed-off-by: Nishant Kumar <www.rajsonu13@gmail.com>

Former-commit-id: a766ea9189219e8ff4450bf86358f6f9c20cc7eb
Change-Id: I202956b4427b4ae7f88df496a108d717a6b8be57
Signed-off-by: minaripenguin <minaripenguin@users.noreply.github.com>
---
 rootdir/init.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index 1e6918d0022c..f29c07bb15af 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -1139,7 +1139,7 @@ on boot
     write /dev/sys/fs/by-name/userdata/cp_interval 200
     write /dev/sys/fs/by-name/userdata/gc_urgent_sleep_time 50
     write /dev/sys/fs/by-name/userdata/iostat_period_ms 1000
-    write /dev/sys/fs/by-name/userdata/iostat_enable 1
+    write /dev/sys/fs/by-name/userdata/iostat_enable 0
 
     # set readahead multiplier for POSIX_FADV_SEQUENTIAL files
     write /dev/sys/fs/by-name/userdata/seq_file_ra_mul 16
